lib/vsc/utils/rest.py

```diff
@@ -172,18 +172,20 @@ def request(self, method, url, body, headers, content_type=None):
         secret_items = ['Authorization', 'X-Auth-Token']
         headers_censored = self.censor_request(secret_items, headers)
 
-        if body and not isinstance(body, str):
-            # censor contents of body to avoid leaking passwords
-            secret_items = ['password']
-            body_censored = self.censor_request(secret_items, body)
-            # serialize body in all cases
-            body = json.dumps(body)
-        else:
-            # assume serialized bodies are already clear of secrets
-            fancylogger.getLogger().debug("Request with pre-serialized body, will not censor secrets")
-            body_censored = body
-
-        fancylogger.getLogger().debug('cli request: %s, %s, %s, %s', method, url, body_censored, headers_censored)
+        body_censored = body
+        if body is not None:
+            if isinstance(body, str):
+                # assume serialized bodies are already clear of secrets
+                logging.debug("Request with pre-serialized body, will not censor secrets")
+                body_censored = body
+            else:
+                # censor contents of body to avoid leaking passwords
+                secret_items = ['password']
```

Comment:
This is already defined on L172?

Reply:
Nevermind, misread

Reply:
it reuses a variable name, but a bit odd to read indeed